### ISSUE TITLE
Creating columns adds an extra space between slide title and content

### ISSUE THREAD

**Issue by m-kru** [andreasKroepelin/polylux#174]
```typ
#slide[
== Title
#columns(2)[
+ airhdl
+ AGWB
+ AutoFPGA
+ Cheby
+ Corsair
+ FPGA Vendors
+ hdl_register
+ II & CII
+ IP_XACT
+ Opentitan Register Tool
+ Register Wizard
+ RgGen
+ SystemRDL
+ vhdMMIO
+ wbgen2
+ Others (2)
]
]
```
Generated slide:
![image](https://github.com/andreasKroepelin/polylux/assets/26227737/c092fd1c-e400-4d61-b01e-d11e6a0dbe17)

The extra space is gone if I remove the `#columns(2)[` command.

**Comment by JamesxX** [comment]
While only a bandaid fix, you could use a negative vertical space after the heading

**Comment by m-kru** [comment]
Yeah, yet another workaround is to `#show heading: set block(below: 1em)`.

**Comment by Enivex** [comment]
I'll copy what I said on the typst repo:

Actually, I don't think this is a bug at all. uncover(2) means that the content is still there on the first slide, just hidden, while only(1) means that the content is only there on the first slide. That means that on the first slide, both texts are present in the item, and will overflow on the next line since they do not fit on a single line

**Comment by Enivex** [comment]
> Yeah, yet another workaround is to `#show heading: set block(below: 1em)`.

I mean, that's not really a work-around, but the intended way to set block spacing. This is not a bug.

**Comment by m-kru** [comment]
So creating columns by default adds an extra space? If so, how can I create columns and inform typst to not add an extra space? Both, negative vertical space and show heading approaches vertically realign the block with columns. However, the realignment value is provided by the user by hand, and it might not be equal to the value of the extra space added automatically by the typst.
